[Improvement] Fix Javadoc issue in task  :lineage:javadoc

### What would you like to be improved?

> Task :lineage:javadoc
/Users/justin/gravitino/lineage/src/main/java/org/apache/gravitino/lineage/LineageDispatcher.java:37: warning - Tag @link: can't find dispatchLineageEvent(RunEvent) in org.apache.gravitino.lineage.LineageDispatcher
1 warning

### How should we improve?

Fix javadoc issue

--- a/lineage/src/main/java/org/apache/gravitino/lineage/LineageDispatcher.java
+++ b/lineage/src/main/java/org/apache/gravitino/lineage/LineageDispatcher.java
@@ -30,7 +30,7 @@ import java.io.Closeable;
  *
  * <ol>
  *   <li>{@link #initialize(LineageConfig)} with required configurations
- *   <li>Repeated calls to {@link #dispatchLineageEvent(RunEvent)}
+ *   <li>Repeated calls to {@link #dispatchLineageEvent(io.openlineage.server.OpenLineage.RunEvent)}
  *   <li>{@link #close()} for resource cleanup
  * </ol>
  */

